frontend/src/components/LandingPage.vue

```diff
@@ -12,30 +13,26 @@
     <v-row class="" align="stretch" justify="space-around">
       <SmallCard :class="smallCardLayout('CCOF')">
```

Comment:
My old me set up "smallCardLayout" to adjust card sizes based on the application status. For new accounts with no applications, the first card should have a column size of 5. But right now, all cards are showing up with the same width.

Reply:
also, all cards should have the same height